libs/resiliency/include/hpx/resiliency/async_replay.hpp

```diff
@@ -142,6 +143,71 @@ namespace hpx { namespace resiliency { namespace experimental {
                     });
             }
 
+            template <std::size_t... Is>
```

Comment:
Can we keep the implementation of the distributed version independent from the local-only version?

Reply:
Like in a different namespace? (like hpx::resiliency::experimental::distributed)

Reply:
Yah, or just in a separate file?

Reply:
I'll add it in a separate file for now. I can't figure out how to get CPOs to work for different namespaces without an argument belonging to that namespace (to trigger ADL).
Let's do it one step at a time ;-)